doc/panes/editor.rst

```diff
@@ -2,23 +2,31 @@
 Editor
 ######
 
-Spyder's multi-language **Editor** integrates a number of powerful tools right out of the box for an easy to use, efficient editing experience.
-The Editor's key features include syntax highlighting (``pygments``); real-time code and style analysis (``pyflakes`` and ``pycodestyle``); on-demand completion, calltips and go-to-definition features (``rope`` and ``jedi``); a function/class browser, horizontal and vertical splitting, and much more.
+Spyder's multi-language **Editor** pane is the key element of the IDE, where you can create, open, and modify source files.
+The Editor offers a variety of core features, such as autocompletion, real-time analysis, syntax highlighting, horizontal and vertical splitting, and much more.
+In addition, it  integrates a number of powerful tools for an easy to use, efficient editing experience.
```

Comment:
```suggestion
In addition, it integrates a number of powerful tools for an easy to use, efficient editing experience.
```

Teeny tiny nit, but It looks like a stray space was retained when this change was manually applied—as a tip, if you use GitHub's "Apply suggestions" feature as I've mentioned, you can be sure that any mistakes are 100% on me :joy: (plus, its a lot faster and easier both for reviewers and yourself :)